lib-ogc/sensorml-core/src/main/java/org/vast/sensorML/PhysicalSystemImpl.java

```diff
@@ -296,4 +309,53 @@ public void setGeometry(AbstractGeometry geom)
         Asserts.checkArgument(geom instanceof Point, "geom must be a Point");
         addPositionAsPoint((Point)geom);
     }
+
+
+    /**
+     * Helper method to parse the geometry from a DataComponent
+     *
+     * @param dataComponent the DataComponent to extract the geometry from
+     * @return the AbstractGeometry if found, null otherwise
+     */
+    private AbstractGeometry parseGeometry(DataComponent dataComponent)
```

Comment:
Can you just rename this method `convertToGeometry` since we are not really parsing data here?